src/vscodeUtils.ts

```diff
@@ -25,7 +25,32 @@ export function getWorkspaceFolder(folderName: string): vscode.WorkspaceFolder |
 // because it also contains Output panes which are considered editors
 // see also https://github.com/microsoft/vscode/issues/58869
 export function getActiveTextEditor(): vscode.TextEditor | undefined {
-  return vscode.window.visibleTextEditors.find((textEditor) => !!textEditor.viewColumn);
+  const activeEditor = vscode.window.visibleTextEditors.find((textEditor) => !!textEditor.viewColumn);
+  return activeEditor;
+}
+
+/*
+  Detects whether this is a Terraform file we can perform operations on
+ */
+export function isTerraformFile(document: vscode.TextDocument): boolean {
```

Comment:
```suggestion
export function isTerraformFile(document?: vscode.TextDocument): boolean {
```

I assume `document` can be `undefined` as we're checking this in line 36?

Reply:
Do we want to allow no parameter to be passed? I thought that's what `?` did if added like this. My intention is to check if undefined, not allow parameter-less invocation

Reply:
Ah, I see. If changing usage to pass a value directly that may be undefined, would have to do this. Changing now